sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -213,19 +213,9 @@ export class ProgramInfoRequestController {
       if (!application) {
         throw new BadRequestException("Application not found.");
       }
-      if (
-        !checkOfferingIntensityMismatch(
-          application.data.howWillYouBeAttendingTheProgram,
-          payload.offeringIntensity,
-        )
-      ) {
-        throw new CustomNamedError(
-          "Offering Intensity does not match the students intensity",
-          OFFERING_INTENSITY_MISMATCH,
-        );
-      }
       // studyStartDate from payload is set as studyStartDate
       let studyStartDate = payload.studyStartDate;
+      let selectedOfferingIntensity = payload.offeringIntensity;
```

Comment:
In my understating there is no scenario where during the PIR complete process there will be no offering available on "payload.selectedOffering". If the offering is not present the API can throw a unprocessable exception.
I am confused with the need of initialization of variables like studyStartDate and selectedOfferingIntensity.

Reply:
I agree. In this scenario, we can avoid the if on line 219 and initialization at line 218.

Reply:
resolving as discussed